addons/web_editor/static/src/js/editor/odoo-editor/src/commands/deleteBackward.js

```diff
@@ -100,7 +99,7 @@ HTMLElement.prototype.oDeleteBackward = function (offset, alreadyMoved = false,
             parentEl.oid !== "root" &&
             parentEl.parentElement &&
             !parentEl.parentElement.isContentEditable &&
-            paragraphRelatedElements.includes(this.tagName) &&
+            this.tagName === 'P' &&
```

Comment:
only 'P' not other block tags?

Reply:
@shsa-odoo  No, only the 'P' tag. otherwise, if the first element inside the banner is a header or any block tag then on backspace it will not convert that tag into a P tag.